lldb/source/Target/LanguageRuntime.cpp

```diff
@@ -289,12 +286,8 @@ void LanguageRuntime::InitializeCommands(CommandObject *parent) {
   if (!parent->IsMultiwordObject())
     return;
 
-  LanguageRuntimeCreateInstance create_callback;
-
   for (uint32_t idx = 0;
-       (create_callback =
-            PluginManager::GetLanguageRuntimeCreateCallbackAtIndex(idx)) !=
-       nullptr;
```

Comment:
I've noticed this unnecessary assignment in the for-loop before, thanks for deleting it.